taca/deliver/cli.py

```diff
@@ -0,0 +1,100 @@
+""" CLI for the deliver subcommand
+"""
+import click
+from ngi_pipeline.utils.communication import mail_analysis
```

Comment:
A lot of the Charon stuff in ngi_pipeline could definitely be moved to that repo. This is probably the best idea, we just didn't have a good enough reason to do it previously. It makes sense though.